.github/dependabot.yml

```diff
@@ -4,13 +4,28 @@
 # https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
 
 version: 2
+registries:
+  maven-github:
+    type: maven-repository
+    url: https://maven.pkg.github.com/minova-afis
+    username: ${{secrets.MAIN_GITHUB_RELEASE_USERNAME}}
+    password: ${{secrets.MAIN_GITHUB_RELEASE_TOKEN}}
 updates:
   - package-ecosystem: "maven"
     directory: "/"
+    registries: "*"
     schedule:
       interval: "daily"
     assignees:
       - "weber-minova"
+  - package-ecosystem: "maven"
+    directory: "/client"
+    registries: "*"
+    schedule:
+      interval: "daily"
+    assignees:
+      - "weber-minova"
+      - "janiak-minova"
```

Comment:
Finde nicht, dass ich bei CAS reingehöre :)

Reply:
> Finde nicht, dass ich bei CAS reingehöre :)

Ist jemand besser geeignet für die `clients` Hierarchie?

Reply:
Wäre mir neu, dass der dependabot Auswirkung auf die Helper hat..

Reply:
Genau, und das client-Projekt ist ja in der Haupt-POM nicht mal eingetragen. Deswegen meine ich ja, im CAS habe ich nichts zu tun :)